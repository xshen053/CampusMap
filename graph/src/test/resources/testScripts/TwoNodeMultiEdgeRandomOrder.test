# Tests adding two nodes, multiple edges to graph
# Random add order

# Create a graph
CreateGraph TNMERO_graph

# Add two nodes
AddNode TNMERO_graph a
AddNode TNMERO_graph b

# Add five edges
AddEdge TNMERO_graph a a AA
AddEdge TNMERO_graph b a BA
AddEdge TNMERO_graph a b AB2
AddEdge TNMERO_graph b b BB
AddEdge TNMERO_graph a b AB1

# Print the nodes in the graph, the outgoing edges from a,
# and outgoing edges from b
ListNodes TNMERO_graph
ListChildren TNMERO_graph a
ListChildren TNMERO_graph b