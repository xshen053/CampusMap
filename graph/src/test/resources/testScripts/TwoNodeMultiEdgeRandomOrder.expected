# Tests adding two nodes, multiple edges to graph
# Random add order

# Create a graph
created graph TNMERO_graph

# Add two nodes
added node a to TNMERO_graph
added node b to TNMERO_graph

# Add five edges
added edge AA from a to a in TNMERO_graph
added edge BA from b to a in TNMERO_graph
added edge AB2 from a to b in TNMERO_graph
added edge BB from b to b in TNMERO_graph
added edge AB1 from a to b in TNMERO_graph

# Print the nodes in the graph, the outgoing edges from a,
# and outgoing edges from b
TNMERO_graph contains: a b
the children of a in TNMERO_graph are: a(AA) b(AB1) b(AB2)
the children of b in TNMERO_graph are: a(BA) b(BB)